import-export-cli/impl/common.go

```diff
@@ -170,3 +171,57 @@ func IncludeMetaFileToZip(sourceZipFile, targetZipFile, metaFile string, metaDat
 	}
 	return nil
 }
+
+//Load the x_meta.yaml file in the provided path and return
+func LoadMetaInfoFromFile(path string) (*utils.MetaData, error) {
+	fileContent, err := GetFileContent(path)
+	if err != nil {
+		return nil, err
+	}
+	metaInfo := &utils.MetaData{}
+	err = yaml.Unmarshal([]byte(fileContent), &metaInfo)
+	if err != nil {
+		return nil, err
+	}
+	return metaInfo, err
+}
+
+//Read the file content from the provided path
+func GetFileContent(path string) (string, error) {
+	r, err := os.Open(path)
+	defer func() {
+		_ = r.Close()
+	}()
+	if err != nil {
+		return "", err
+	}
+	data, err := ioutil.ReadAll(r)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+//Find the path for a file matching the provided patter. If that file is not found in the root directory, an empty string
+//will be returned
+func GetFileLocationFromPattern(root, pattern string) (match string, err error) {
+	err = filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() {
+			return nil
+		}
+		if matched, err := filepath.Match(pattern, filepath.Base(path)); err != nil {
+			return err
+		} else if matched {
+			match, err = filepath.Abs(path)
+			if err != nil {
+				return err
+			}
+			return io.EOF
+		}
+		return nil
+	})
+	return
```

Comment:
This function doesn't seem to return the matched file name as expected

Reply:
The match variable is already defined in the return arguments of the function.

Reply:
Since this has a function within a function with quite a few conditional statements, combining it with this particular style tends to be a bit confusing to the reader. So it might be better to go with the more traditional syntax to make it clear. Will reduce the chance of bugs if we modify this in the future.

Reply:
Fixed with 9a12cbd